Rest docs has missing swagger requirement

A new installation of openHAB 2.4.0-SNAPSHOT # 1300 fails to start with the following error message.

`[ERROR] [.core.karaf.internal.FeatureInstaller] - Failed installing 'openhab-binding-hue, openhab-binding-yahooweather, openhab-misc-restdocs, openhab-binding-avmfritz, openhab-binding-wemo, openhab-binding-sonos, openhab-ui-habpanel, openhab-misc-hueemulation, openhab-binding-ipp, openhab-persistence-rrd4j, openhab-ui-basic, openhab-transformation-map, openhab-binding-ntp, openhab-misc-homekit, openhab-binding-astro, openhab-ui-paper': Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=openhab-misc-restdocs; type=karaf.feature; version="[2.4.0.SNAPSHOT,2.4.0.SNAPSHOT]"; filter:="(&(osgi.identity=openhab-misc-restdocs)(type=karaf.feature)(version>=2.4.0.SNAPSHOT)(version<=2.4.0.SNAPSHOT))" [caused by: Unable to resolve openhab-misc-restdocs/2.4.0.SNAPSHOT: missing requirement [openhab-misc-restdocs/2.4.0.SNAPSHOT] esh.tp; filter:="(feature=jax-rs-provider-swagger)"]`

Should the io.swagger.annotations dependency of org.openhab.io.rest.docs be made optional as well like you did in https://github.com/openhab/org.openhab.ui.habpanel/pull/305 @kaikreuzer ?

--- a/features/openhab-core/src/main/feature/feature.xml
+++ b/features/openhab-core/src/main/feature/feature.xml
@@ -98,20 +98,10 @@
         <feature>esh-io-transport-upnp</feature>
     </feature>
 
-    <feature name="openhab-misc-ruleengine" description="Rule Engine (Experimental)" version="${project.version}">
+    <feature name="openhab-misc-restdocs" description="REST Documentation" version="${project.version}">
         <feature>openhab-runtime-base</feature>
-        <feature>esh-automation-api</feature>
-        <feature>esh-automation-commands</feature>
-        <feature>esh-automation-core</feature>
-        <feature>esh-automation-module-core</feature>
-        <feature>esh-automation-module-media</feature>
-        <feature>esh-automation-module-script</feature>
-        <feature>esh-automation-module-script-defaultscope</feature>
-        <feature>esh-automation-module-script-rulesupport</feature>
-        <feature>esh-automation-module-timer</feature>
-        <feature>esh-automation-parser-gson</feature>
-        <feature>esh-automation-providers</feature>
-        <feature>esh-automation-rest</feature>
+        <feature>esh.tp-swagger-jax-rs-provider</feature>
+        <bundle>mvn:org.openhab.core/org.openhab.io.rest.docs/${project.version}</bundle>
     </feature>
 
     <feature name="openhab-ui-homebuilder" description="Home Builder" version="${project.version}">

